xlators/cluster/afr/src/afr-common.c

```diff
@@ -1572,7 +1572,7 @@ afr_set_split_brain_choice(int ret, call_frame_t *frame, void *opaque)
     return 0;
 }
 
-int
+static int
 afr_accused_fill(xlator_t *this, dict_t *xdata, unsigned char *accused,
```

Comment:
nit: same as previously, instead of 'this' - you can pass 'priv'. Worth doing in a different patch though.